docs/src/main/asciidoc/getting-started-reactive.adoc

```diff
@@ -160,6 +169,13 @@ But enough talking, let's get our hands dirty!
 
 == Bootstrapping the project
 
+There are several ways to implement reactive application with Quarkus.
+In this guide we are going to use RESTEasy Reactive, an implementation of RESTEasy benefiting from the Quarkus reactive engine.
+By default, it invokes the HTTP endpoint on the I/O thread.
+
+IMPORTANT: While it's possible to use _traditional_ RESTEasy, you would need to add the `quarkus-resteasy-mutiny` extension, and the method will still be invoked on a _worker_ thread.
+So, while it would use reactive programming, it would still requires worker threads, which defeat the purpose.
```

Comment:
💯 . This is something that people have not yet grokked and something which I need to try and convey in a blog post :)

Reply:
Yes! I didn't want to add a specific image in this guide, but definitely something for a blog post.